images/Dockerfile

```
@@ -27,6 +27,17 @@ RUN dnf install -y \
     && dnf clean all
 
+# Install Claude Code CLI via official RPM repository
+RUN tee /etc/yum.repos.d/claude-code.repo <<'EOF'
+[claude-code]
+name=Claude Code
+baseurl=https://downloads.claude.ai/claude-code/rpm/stable
+enabled=1
+gpgcheck=1
+gpgkey=https://downloads.claude.ai/keys/claude-code.asc
+EOF
+RUN dnf install -y claude-code && dnf clean all
```

Hadolint (2.14.0)

[error] unexpected 'R'
expecting a new line followed by the next instruction

(DL1000)